### DISCUSSION TITLE
Servesless function has crashed for Flask Application.

### DISCUSSION THREAD

**Discussion by arteevraina** [vercel/community#2217] [Help] [answered]
### Summary

I have been trying to deploy my flask app to vercel. I have successfully deployed it many times before and it actually worked.
But, I have been facing this issue for now.
<img width="678" alt="Screenshot 2023-04-28 at 12 21 10 AM" src="https://user-images.githubusercontent.com/43968121/234963226-4984f128-594e-4976-9844-8a1b922ac758.png">

```
[ERROR] Runtime.ImportModuleError: Unable to import module 'vc__handler__python': cannot import name 'JSONEncoder' from 'flask.json' (/var/task/flask/json/__init__.py)
Traceback (most recent call last):
```

There is no such module in my actual code.

This is how my `vercel.json` looks like:

```
{
    "builds": [
      {
        "src": "server.py",
        "use": "@vercel/python"
      }
    ],
    "routes": [
      {
        "src": "/(.*)",
        "dest": "/server.py"
      }
    ]
  }
  
```

### Additional information

Link to my flask app: https://github.com/arteevraina/registry/tree/main/flask

**Comment by lunasupps** [comment]
Hi, did you eventually manage to fix this issue? Im struggeling with the same thing

**Reply by arteevraina** [reply, marked as answer]
Yes, you can try to deploy it locally ``vercel dev`` or some command like that to serve your website on localhost.
After that, you can look into logs which file is failing.
For me, it was the flask version which had some issues so I had to rollback to a previous version to fix that.

**Reply by lunasupps** [reply]
Oh thank you! I'll try that

**Reply by akechi0523** [reply]
hi,
I've successfully deployed the flask backend locally using vercel dev, but it still doesn't work with vercel.
Same error.
Could you please help me?
thank you
